server/graphql/v2/interface/Account.js

```diff
@@ -300,20 +300,56 @@ const accountFieldsDefinition = () => ({
       ...CollectionArgs,
       onlyPublishedUpdates: { type: GraphQLBoolean },
       onlyChangelogUpdates: { type: GraphQLBoolean },
+      sortBy: { type: GraphQLString },
+      searchTerm: { type: GraphQLString },
     },
-    async resolve(collective, { limit, offset, onlyPublishedUpdates, onlyChangelogUpdates }) {
+    async resolve(collective, { limit, offset, onlyPublishedUpdates, onlyChangelogUpdates, sortBy, searchTerm }) {
       let where = {
         CollectiveId: collective.id,
+        [Op.and]: [],
       };
+      const include = [{ association: 'fromCollective', required: true, attributes: [] }];
```

Comment:
This `include` does not seem necessary if there's no `searchTerm`

Reply:
Thanks. I've refactored it. 👍🏽